pkg/utils/time.go

```diff
@@ -30,3 +30,8 @@ func CurrentEpochSecsInInt() int {
 func CurrentEpochNanoSecsInInt64() int64 {
 	return time.Now().UnixNano()
 }
+
+// SecsToNanoSecsInInt64 converts a value from secs to nanoseconds.
+func SecsToNanoSecsInInt64(secs int64) int64 {
```

Comment:
is this fuction actually used anywhere?

Reply:
yes, i see it's missing where the timestamp gets set.  I can add that back.

Reply:
but wait, it doesn't make sense to put it as nano seconds if we are just multiplying by 10^9. we should just adapt to the time unit of the block timestamp

Reply:
The block timestamp is just in secs from epoch.  I just adapted it here bc we are storing as nanoseconds throughout.

Reply:
as per discussion in slack, will change these timestamps to secs from epoch